common/djangoapps/enrollment/views.py

```diff
@@ -283,7 +283,7 @@ class EnrollmentCourseRosterView(APIView, ApiKeyPermissionMixIn):
     """
     Read roster for a particular course. (contains PII)
     """
-    authentication_classes = EnrollmentCrossDomainSessionAuth,
+    authentication_classes = OAuth2AuthenticationAllowInactiveUser,
```

Comment:
@jbassen What motivated this?

My concern is the `AllowInactiveUser` bit.
My quick read seems like that would allow inactive users to access the API,
which we shouldn't want/need; instructors should always activate their accounts.

Can you provide some insight?